packages/jbr/README.md

```diff
@@ -287,6 +287,78 @@ $ jbr run -d docker-options.json
 
 More configuration options can be found at https://github.com/apocas/dockerode#getting-started
 
+### Querying local files in Comunica experiment
```

Comment:
Nice docs! But I would suggest moving it into the README of `hook-sparql-endpoint-comunica`, as it only applies there.

Reply:
Also done, didn't know there was a separate readme for hook-sparql-endpoint-comunica